api/src/main/java/com/codeforcommunity/dto/team/InviteUsersRequest.java

```diff
@@ -0,0 +1,38 @@
+package com.codeforcommunity.dto.team;
+
+import com.codeforcommunity.dto.ApiDto;
+import com.codeforcommunity.exceptions.HandledException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class InviteUsersRequest extends ApiDto {
+
+  private List<InviteContact> users;
+
+  public InviteUsersRequest(List<InviteContact> users) {
+    this.users = users;
+  }
+
+  public InviteUsersRequest() {}
+
+  public List<InviteContact> getUsers() {
+    return users;
+  }
+
+  public void setUsers(List<InviteContact> users) {
+    this.users = users;
+  }
+
+  @Override
+  public List<String> validateFields(String fieldPrefix) throws HandledException {
+    String fieldName = fieldPrefix + "invite_user_request.";
+    List<String> fields = new ArrayList<>();
+
+    if (users == null) {
+      fields.add(fieldName + "users");
+    } else {
+      users.forEach(user -> user.validateFields(fieldPrefix));
```

Comment:
Make sure to update the fieldPrefix so it's clear the error happened inside a user object: `validateFields(fieldPrefix + "users.")` and to add the result of each validateFields call to the "fields" object so that it's returned from this method `fields.addAll(user.validateFields(...))`